### PR TITLE
Transfers Section

### PR DESCRIPTION
**Pull request by aravind-segu**

### CODE REVIEW

File: content/transfers.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <span class="graph-title">Proportion of students who transferred.</span>
+    <div id="students-transferred"></div>
+</div>
+
+<div class="vbx2"></div>
+
+<h3> Term Switched Out </h3>
+<p class="half">
+<span class="big">
+    Students started switching out of the program after completing second year
```
Comment (roxelo) [inline content/transfers.html]: I think the reason why we are not seeing any time earlier is that we only asked people who at least spent 2 years in the program to complete the survey. Maybe we should incorporate the transfer info from the 2019 se board too for a more complete picture https://patricklam.ca/se/se-board-2019-dec/

**Review by roxelo — COMMENTED** [review]
LGTM, only thing that we may want to change a bit is the data around when people transferred since people did also transferred in second year.

**Comment by patricklam** [comment]
We have had transfers into ECE (about one per 5 years) and various random
other programs at some times but maybe not your class. I may have mentioned
that in one of my reports.

**Comment by roxelo** [comment]
LGTM

**Pull request merged**